src/core/flameshotdaemon.cpp

```diff
@@ -151,10 +151,7 @@ void FlameshotDaemon::copyToClipboard(const QString& text,
     auto m = createMethodCall(QStringLiteral("attachTextToClipboard"));
 
     m << text << notification;
-
-    QDBusConnection sessionBus = QDBusConnection::sessionBus();
-    checkDBusConnection(sessionBus);
-    sessionBus.call(m);
+    call(m);
```

Comment:
Why was this change made?

Reply:
It has nothing to do with exit code, but I find these 3 lines are equivalent to `FlameshotDaemon::call`, just change it by the way. Did I make something wrong?

Reply:
It's probably fine, but it would be better to split it into 2 PR's or commits where 1 commit is the new feature, and 1 is the code cleanup.

I have had to bisect the code quite a bit lately, and having separate thoughts broken out to separate commits helps with that.

Reply:
I have rewrote the commit history to separate code clean against with the new feature. Hope it helps later. Feel free to post any further issues if you have ;)